src/System.Numerics.Vectors.WindowsRuntime/src/System/Numerics/VectorExtensions.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Windows.Foundation;
+
+namespace System.Numerics
+{
+    /// <summary>
+    /// Contains conversions from Vector2 to Windows.Foundation.Point and Size, which are all structurally identical.
```

Comment:
"from Vector2 to Windows.Foundation.Point and Size"... it also contains the other direction